### PR TITLE
[Notifier] Create `DesktopChannel` and `JoliNotif` bridge

### PR DESCRIPTION
**Pull request by ahmedghanem00**
Updating docs according to symfony/symfony#57683

Fixes https://github.com/symfony/symfony-docs/issues/20168

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
This looks correct to me ... but I'm not an expert here, so I'll wait for more approvals before merging. Thanks!

**Review by OskarStark — APPROVED** [review]
Looks good to me, but a versioned directive is missing for 7.2

**Comment by ahmedghanem00** [comment]
> Looks good to me, but a versioned directive is missing for 7.2

Can you provide an example for this directive? :)

File: notifier.rst [hunk 1]
```diff
@@ -623,6 +624,71 @@ configure the ``texter_transports``:
...
+
+        // config/packages/notifier.php
+        use Symfony\Config\FrameworkConfig;
+
+        return static function (FrameworkConfig $framework): void {
+            $framework->notifier()
+                ->texterTransport('jolinotif', env('JOLINOTIF'))
+            ;
+        };
+
```
Comment (OskarStark) [inline notifier.rst:695]:
```suggestion

.. versionadded:: 7.2

    The `Desktop` channel was introduced in Symfony 7.2.

```

**Comment by OskarStark** [comment]
> Can you provide an example for this directive? :)

Done sorry :-)

File: notifier.rst [hunk 2]
```diff
@@ -623,6 +624,75 @@ configure the ``texter_transports``:
...
+The desktop channel is used to display desktop notifications on the same host machine using
+:class:`Symfony\\Component\\Notifier\\Texter` classes. Currently, Symfony
+is integrated with the following providers:
+
+===============  ====================================  ==============================================================================
+Provider         Package                               DSN
+===============  ====================================  ==============================================================================
+`JoliNotif`_     ``symfony/joli-notif-notifier``       ``jolinotif://default``
+===============  ====================================  ==============================================================================
+
```
Comment (OskarStark) [inline notifier.rst:641]:
```suggestion

.. versionadded: 7.2

    The JoliNotif bridge was introduced in Symfony 7.2.

```

File: notifier.rst [hunk 3]
```diff
@@ -623,6 +624,75 @@ configure the ``texter_transports``:
...
+        return static function (FrameworkConfig $framework): void {
+            $framework->notifier()
+                ->texterTransport('jolinotif', env('JOLINOTIF'))
+            ;
+        };
+
+.. versionadded:: 7.2
+
+    The `Desktop` channel was introduced in Symfony 7.2.
```
Comment (OskarStark) [inline notifier.rst]:
```suggestion
    The ``Desktop`` channel was introduced in Symfony 7.2.
```

**Review by OskarStark — APPROVED** [review]
Good to merge after my two comments

**Comment by javiereguiluz** [comment]
Merged! Ahmed, thanks a lot for this contribution.

**Pull request merged**